gmlc/networking/TcpServer.cpp

```diff
@@ -254,15 +340,17 @@ void TcpServer::handle_accept(
     TcpConnection::pointer new_connection)
 {
     /*setting linger to 1 second*/
-    asio::socket_base::linger optionLinger(true, 0);
-    new_connection->socket().set_option(optionLinger);
-    new_connection->socket().set_option(asio::ip::tcp::no_delay(true));
+    new_connection->socket()->set_option_linger(true, 0);
+    new_connection->socket()->set_option_no_delay(true);
```

Comment:
Did you look in HELICS if we were using any other options for sockets?

Reply:
We aren't using any other options for sockets in HELICS (for standalone asio). The helicsWebServer stuff using boost::asio is using some websocket/beast options.